src/Components/Charts/GLD.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from "react";
+import { PieChart, Pie, Legend, Tooltip, Cell } from "recharts";
+
+const data01 = [{ name: "Gold", value: 100 }];
+
+const COLORS = [
+  "#FFCD1C",
```

Comment:
Looks like we use always the same colors, this cost could move to a theme folder and exported to avoid repetition.